Packages/com.unity.multiplayer.samples.coop/Utilities/SceneManagement/NetworkedLoadingProgressTracker.cs

```diff
@@ -0,0 +1,15 @@
+using Unity.Netcode;
+
+namespace Unity.Multiplayer.Samples.Utilities
+{
+    /// <summary>
+    /// Simple object that keeps track of the scene loading progress of a specific instance
+    /// </summary>
+    public class NetworkedLoadingProgressTracker: NetworkBehaviour
```

Comment:
Our overall discussion over using owner netvars for client state tracking should be documented. We have some doc on client driven stuff, let's add something there, how it needs a NO per player for it to work.